src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/HoverInfoService.cs

```diff
@@ -21,19 +24,18 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Hover;
 
+[Export(typeof(IRazorDocumentMappingService)), Shared]
```

Comment:
Should this have been `IHoverInfoService`?

Reply:
Good catch!

Reply:
yes, and points out a huge flaw in my testing that I didn't verify which endpoint was calling the business logic